packages/common/log.js

```diff
@@ -1,36 +1,79 @@
 'use strict';
 
-const tagged = (log, tag, thisObj) =>
-  log.bind(thisObj || console, `[${tag}]`);
-
-const use = (log, error = null) =>
-  Object.assign({}, module.exports, {
-    log: log,
-    info: tagged(log, 'INFO'),
-    warn: tagged(log, 'WARN'),
-    debug: tagged(log, 'DEBUG'),
-    error: tagged(error || log, 'ERROR')
-  });
-
-const logs = use(console.log, console.error); // eslint-disable-line no-console
-module.exports = Object.assign({}, logs);
-
-const reset = () => {
-  Object.assign(module.exports, logs);
-};
-
-module.exports = {
-  use: use,
-  tagged: tagged,
-  mute: (...levels) => {
-    reset();
-    for (const level of levels) {
-      module.exports[level] = () => null;
+/**
+ * Constructs JSON to log
+ *
+ * @param {string} level - type of log (info, error, debug, warn)
+ * @param {string} args - Message to log and any other information
+ * @returns {JSON} - the JSON to be logged
+ */
+function log(level, args) {
+  const executions = process.env.EXECUTIONS; //set in handler from cumulus_meta
+  const sender = process.env.SENDER; //set in handler from AWS context
+  const time = new Date();
+  var message = '';
+
+  const output = {
+    executions: executions,
+    timestamp: time.toISOString(),
+    sender: sender,
+    level: level
+  };
+  for (const arg of args) {
+    if ((typeof arg) === 'string') message += arg;
+    else {
+      for (const key in arg) {
+        output[key] = arg[key];
+      }
     }
-  },
-  unmute: () => {
-    reset();
   }
-};
 
-reset();
+  output.message = message;
+
+  console.log(output);
+}
+
+/**
+ * Logs the message
+ *
+ * @param {string} args - Includes message and any other information to log
+ * @returns {JSON} - the JSON to be logged
+ */
+function info(...args) {
+  return log('info', args);
+}
+
+/**
+ * Logs the error
+ *
+ * @param {Object} args - Includes error and any other information to log
+ * @returns {JSON} - the JSON to be logged
+ */
+function error(...args) {
+  log('error', args);
+}
+
+/**
+ * Logs the debugger messsages
+ *
+ * @param {Object} args - Includes debugger message and any other information to log
+ * @returns {JSON} - the JSON to be logged
+ */
+function debug(...args) {
+  log('debug', args);
+}
+
+/**
+ * Logs the Warning messsage
+ *
+ * @param {Object} args - Includes Warn message and any other information to log
+ * @returns {JSON} - the JSON to be logged
+ */
+function warn(...args) {
+  log('warn', args);
+}
+
+module.exports.info = info;
+module.exports.error = error;
+module.exports.debug = debug;
+module.exports.warn = warn;
```

Comment:
could you add the same levels as the Java logger? https://github.com/cumulus-nasa/cumulus-message-adapter-java/blob/cc6258aecf8b5eb8c592670f8850050f673c2006/message_parser/src/main/java/cumulus_message_adapter/message_parser/AdapterLogger.java#L25

Reply:
Added all of them